README.md

```diff
@@ -52,10 +52,12 @@ You can configure the plugin via the [`COMPOSER_HOME/config.json`](https://getco
 {
     "config": {
         "sllh-composer-versions-check": {
+            "ignore-sub-dependencies": false,
             "show-links": false
         }
     }
 }
 ```
 
+* `ignore-sub-dependencies`: Shows only outdated package required in composer.json.
```

Comment:
```suggestion
* `ignore-sub-dependencies`: Shows only outdated root packages.
```

We may also rename the option to something like `root-packages-only` to be consistent with the code naming. What do you think?

Reply:
Sounds good, I'll do it